spec/controllers/results_controller_spec.rb

```diff
@@ -11,53 +11,96 @@
 
       get :new, game_id: game
 
-      assigns(:result).should_not be_nil
+      expect(assigns(:result)).not_to be_nil
     end
 
     it "exposes the game" do
       game = FactoryGirl.create(:game)
 
       get :new, game_id: game
 
-      assigns(:game).should == game
+      expect(assigns(:game)).to eq(game)
     end
   end
 
   describe "create" do
-    context "with valid params" do
-      it "creates a new result with the given players" do
+    context "when defeated the opponent" do
+      it "creates a new result with the current player as the winner" do
         game = FactoryGirl.create(:game, results: [])
-        player_1 = FactoryGirl.create(:player)
-        player_2 = FactoryGirl.create(:player)
+        opponent = FactoryGirl.create(:player)
+        current_player = FactoryGirl.create(:player, email: TEST_EMAIL)
```

Comment:
In `spec_helper` we have: 
```ruby
sign_in FactoryGirl.create(:user, email: TEST_EMAIL)
```
So here we need to create a `player` with the same email address so that `Player.find_by(email: email)` can find it.